docs/install/prerequisites.md

```diff
@@ -2,10 +2,7 @@
 
 Portable (CO:RE) option:
 - Linux kernel version >= 4.18
-- `CONFIG_DEBUG_INFO_BTF` Linux compile option enabled (only needed for running, not compilation)
-- libc, and the libraries: libelf, zlib
-- GNU Make >= 4.3
-- clang >= 11
+- BTF enabled
```

Comment:
I suggest to add the same detection stanza: You can manually detect if your environments supports it by checking if the following file exists on your machine: path/to/vmlinux or consult the following documentation: https://github.com/libbpf/libbpf#bpf-co-re-compile-once--run-everywhere.